packages/cursorless-vscode/src/ide/vscode/VscodeFileSystem.ts

```diff
@@ -4,11 +4,13 @@ import {
   PathChangeListener,
   RunMode,
 } from "@cursorless/common";
+import * as fs from "fs/promises";
 import { isAbsolute, join } from "path";
 import * as vscode from "vscode";
 
 export class VscodeFileSystem implements FileSystem {
   public readonly cursorlessTalonStateJsonPath: string;
+  readonly cursorlessTalonStateJsonDirPath: string;
```

Comment:
This isn't necessary if we go with https://github.com/cursorless-dev/cursorless/pull/2481 so that one should probably merge first